internal/alb/tg/targets.go

```diff
@@ -157,3 +164,32 @@ func tdsString(tds []*elbv2.TargetDescription) string {
 	}
 	return strings.Join(s, ", ")
 }
+
+func populateTargetAZ(ctx context.Context, c aws.CloudAPI, a []*elbv2.TargetDescription) error {
```

Comment:
nitpick:
I prefer func (c *targetsController) populateTargetAZ(ctx context.Context, a []*elbv2.TargetDescription) {
    //use c.cloud when reference cloud 😄 
}

Reply:
Added `targetsController` as a receiver for `populateTargetAZ` and moved the function definition up with the other `targetsController` functions.